docs/explanation/upgrade-groups.rst

```diff
@@ -0,0 +1,75 @@
+==============
+Upgrade Groups
+==============
+
+In the process of planning or executing an upgrade for an OpenStack cloud, users have the
+capability to target a specific group of applications based on their services responsibilities.
+
+**Note:** COU will upgrade **openstack-dashboard** and **octavia** at the end of the
+**control-plane** upgrade (before upgrading **control-plane** subordinate applications
+and **data-plane** services) due to the desired upgrade group splits. This is slightly
+different from the `upstream upgrade documentation`_.
+
+Control Plane
+-------------
+
+The **control-plane** includes services tasked with making decisions related to data management,
+routing, and processing. Services considered as **control-plane** in OpenStack and in scope
+of COU are (following their upgrade order):
+
+- rabbitmq-server
+- ceph-mon
+- keystone
+- aodh
+- barbican
+- ceilometer
+- ceph-fs
+- ceph-radosgw
+- cinder
+- designate
+- designate-bind
+- glance
+- gnocchi
+- heat
+- manila
+- manila-ganesha
+- neutron-api
+- neutron-gateway
+- ovn-dedicated-chassis
+- ovn-central
+- placement
+- nova-cloud-controller
+- openstack-dashboard
+- octavia
+- **control-plane** subordinate applications
```

Comment:
before the subordinates there are principal charms not listed on the documentation. E.g: `mysql-innodb-cluster`

```suggestion
- **other principal applications** not listed in the upgrade order
- **control-plane** subordinate applications
```